drivers/volume/portworx/portworx.go

```diff
@@ -49,7 +49,7 @@ const (
 	maintenanceWaitTimeout           = 2 * time.Minute
 	inspectVolumeTimeout             = 10 * time.Second
 	inspectVolumeRetryInterval       = 2 * time.Second
-	validateDeleteVolumeTimeout      = 3 * time.Minute
+	validateDeleteVolumeTimeout      = 5 * time.Minute
```

Comment:
why is the timeout bumped? A PVC should get deleted in 3 minutes.

Reply:
sometime it's taking longer than that, actually no error for that

Reply:
actually this piece of code belong to another ticket I only put in here to test three problems at once

Reply:
revert this timeout. If deletes are timing out, we need to root cause the actual issue in PX.

Reply:
ok, I might need some help on that since I haven't found any error in the logs

Reply:
Sure, I can help take a look if you have a cluster where PVC delete took more than 3 minutes. First place to look is the controller manager logs which will have the errors reported on the PVC delete call to Portworx